invisible_cities/cities/detsim_simulate_electrons_test.py

```diff
@@ -42,14 +44,21 @@ def MChits_and_detsim_electron_simulation_params(nexusMC_filename):
     transverse_diffusion   = 1.00 * units.mm / units.cm**0.5
     longitudinal_diffusion = 0.30 * units.mm / units.cm**0.5
 
-    return xs, ys, zs, energies,\
-           wi, fano_factor, lifetime, drift_velocity,\
-           transverse_diffusion, longitudinal_diffusion
+
+    typename = "MChits_and_detsim_params"
+    field_name = ["xs", "ys", "zs", "energies", "wi", "fano_factor", "lifetime",
+                  "drift_velocity", "transverse_diffusion", "longitudinal_diffusion"]
+
+    nt = namedtuple(typename, field_name)
+    return nt(xs, ys, zs, energies, wi, fano_factor, lifetime,
+              drift_velocity, transverse_diffusion, longitudinal_diffusion)
 
 
-def test_generate_ionization_electrons(MChits_and_detsim_electron_simulation_params):
-    _, _, _, energies,\
-    wi, fano_factor, *_ = MChits_and_detsim_electron_simulation_params
+def test_generate_ionization_electrons(MChits_and_detsim_params):
+
+    energies    = MChits_and_detsim_params.energies
+    wi          = MChits_and_detsim_params.wi
+    fano_factor = MChits_and_detsim_params.fano_factor
```

Comment:
That's fine ... but in some cases you might consider an approach along these lines:

```python
f = MChits_and_detsim_params
secondary_electrons = generate_ionization_electrons(f.energies, f.wi, f.fano_factor)
```

Here I picked `f` for **F**ixture, but something else might be more appropriate depending on context.